helm-charts/splunk-otel-collector/templates/daemonset.yaml

```diff
@@ -177,6 +177,14 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+          - name: MY_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: MY_NODE_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
```

Comment:
Why do we need to add these if the data already available under `K8S_NAMESPACE` and `K8S_NODE_NAME`?

Reply:
Hi @dmitryax 
As mention in [OTL-951](https://signalfuse.atlassian.net/browse/OTL-951) when we are trying to set up the Kubernetes-events plugin using the Otel collector we can observe the following error in `otel-collector` container
<img width="1773" alt="Screenshot 2021-09-01 at 9 21 30 AM" src="https://user-images.githubusercontent.com/88535006/131610420-71cb2270-da32-472f-aff7-d6d4519e7d3e.png">
<img width="1773" alt="Screenshot 2021-09-01 at 9 20 07 AM" src="https://user-images.githubusercontent.com/88535006/131610435-f6e44bf4-f36d-4101-8d85-1957d1965070.png">
Because Kubernetes-events monitor is requesting  `MY_NAMESPACE`, `MY_NODE_NAME` environmental variables https://github.com/signalfx/signalfx-agent/blob/main/pkg/monitors/kubernetes/leadership/leadership.go#L73